.github/workflows/ci.yml

```diff
@@ -255,10 +255,11 @@ jobs:
     steps:
       - name: Checkout code
         uses: actions/checkout@v3
-      - name: Use Node.js 14.17
+      - name: Use Node.js latest
         uses: actions/setup-node@v3
         with:
-          node-version: "14.17" # Node.js 14.17 is the first LTS supported by Babel 8
+          node-version: latest
+          check-latest: true
```

Comment:
I'm worried this will cause some issues to go undetected.

Not related to this PR: Do we have a minimum node version requirement for the repository (build and development)

Reply:
> Do we have a minimum node version requirement for the repository (build and development)

Currently no. Personally I build / develop on latest Node and upgrade my environment every week. I believe using latest node is important:
- If in the future any node update break our build pipeline, we can always fallback to an older version temporarily. (IIRC in the past this scenario only happened in major upgrade)
- Pinning node version will deprive us the benefit of latest build tools, e.g. Jest / Yarn will drop unmaintained node versions

Reply:
Makes sense. I also currently always keep the latest major version.